Documentation/common-issues.md

```diff
@@ -325,6 +330,18 @@ $ kubectl -n rook-ceph describe pod -l mon=rook-ceph-mon0
 ...
 ```
 
+See the solution in the next section regarding cleaning up the `dataDirHostPath` on the nodes.
+
+### Three mons named a, d, and f
```

Comment:
@leseb the challenge is that the residual files will be on the host where the daemon is, and the operator won't be able to see them. The only way I can see to detect the residual files is by running a rook init container as before. I'm not sure it's worth putting it back if this is the only reason.

Reply:
If this enhances the user experience then I believe it's worth putting back. I suspect that many people are getting false positive because they forget to purge residual files.

Reply:
Yes, definitely need to consider that, will discuss separately from this PR

Reply:
I had a thought the other day that when the operator creates a mon on a node, it could run a job on that node to erase the contents of the data dir which the upcoming pod is going to use. This would solve the issue, and it still leaves all control to the operator.